canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -58,6 +58,13 @@ <h4 class="list-group-item-heading">Find Course Info</h4>
           </a>
         </div>
 
+        <div class="list-group col-md-4">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}" class="list-group-item active">
+              <h4 class="list-group-item-heading">Cross-list Courses</h4>
+              <p class="list-group-item-text">Set up and modify cross-listings <br>&nbsp;</p>
```

Comment:
I guess we could. It has been placed there to compensate for the fact that other blue buttons (like Conclude Courses), have two lines of descriptive text. Without this extra space, the blue buttons of Find Course Info and Cross-List Courses will appear inconsistent in height. I could look into a style with a max (or min) height value.
P.S. Please remember that he UI code I submit to GitHub is just a recommendation for a UI code, not the ultimate solution.